### ISSUE TITLE
Does osticket support google mail authentication?

### ISSUE THREAD

**Issue by thaygiaoth** [osTicket/osTicket#6002]
Hello all

I want to know osticket support google authentication in core or plugin

My company is really really need this feature

Thank you very much!

**Comment by JediKev** [comment]
@thaygiaoth 

The title makes this a little confusing.

Are you asking if you can configure Google SSO/OAuth for Agents/Users? If so, not at this time.

Are you asking if you can configure 2FA for Agents via Google Authenticator app? If so, yes. ([see here](https://docs.osticket.com/en/latest/Plugins/Two%20Factor%20Authentication.html))

Are you asking if you can configure a system email to authenticate IMAP/POP/SMTP via Google (Gmail)? If so, yes. ([see here](https://docs.osticket.com/en/latest/Admin/Emails/Emails.html))

Cheers.

**Comment by itviet2021** [comment]
@JediKev

Are you asking if you can configure Google SSO/OAuth for Agents/Users? If so, not at this time.

Yes it is

Please support it

Thank you very much for your rapid reply

**Comment by itviet2021** [comment]
@JediKev

I found auth-oauth in an osticket repo, 7 years ago

https://github.com/osTicket/osTicket-plugins

**Comment by JediKev** [comment]
@itviet2021 

Yes, but as you mentioned it hasn’t been updated in 7 years and may no longer function correctly. This plugin was never officially released.

Cheers.
